### PR TITLE
Upgraded templates, added rake console task

### PR DESCRIPTION
**Pull request by mxmCherry**
Also copied code from https://github.com/bsm/grape-app/pull/3 (just to make it happen faster)

### CODE REVIEW

**Comment by mxmCherry** [comment]
Uh, tests failed...

```
$ rvm use 2.5.1 --install --binary --fuzzy
...
$ bundle exec rake
bundler: failed to load command: rake (/home/travis/build/bsm/grape-app/vendor/bundle/ruby/2.5.0/bin/rake)
```

Not sure why it's trying to get rake from 2.5.0

@dim would appreciate help on fixing the build. Though, 2.5.1 seems to be excessive, 2.5.0 is enough (no need to have two 2.5.x versions)

File: lib/grape/app/templates/Rakefile [hunk 1]
```diff
@@ -1,6 +1,22 @@
...
+begin
+  require 'rspec/core/rake_task'
+  RSpec::Core::RakeTask.new(:spec)
+  Rake::Task['spec'].enhance ['db:test:prepare']
+rescue LoadError
+  # so it won't fail in production env, where rspec is usually not installed
+end
+
+desc 'Launch console'
+task console: :environment do
```
Comment (dim) [inline lib/grape/app/templates/Rakefile]: can you please move that one to grape/app/tasks/console.rake?

File: lib/grape/app/templates/Rakefile [hunk 2]
```diff
@@ -1,6 +1,22 @@
 require 'bundler/setup'
-require 'rspec/core/rake_task'
 require 'grape/app/tasks'
 
-RSpec::Core::RakeTask.new(:spec)
+begin
+  require 'rspec/core/rake_task'
+  RSpec::Core::RakeTask.new(:spec)
+  Rake::Task['spec'].enhance ['db:test:prepare']
```
Comment (dim) [inline lib/grape/app/templates/Rakefile]: please remove that line. we do have "maintain schema" in spec helper, it should work. if not, we should not workaround like this

**Comment by dim** [comment]
would also be good to replace things rubocop complains about. you should add rubocop to the dev-dependencies of that gem and run it. so far, I found:

1. replace `__FILE__` with `__dir__` (and remove one set of `../`) e.g. in  `config/environment.rb` and in `spec/spec_helper.rb`
2. remove spaces on top of bottom of config/environments/*.rb
3. there is a `%w|...|` somewhere, should be `%w[...]`

File: lib/grape/app/templates/Rakefile (hunk 2 above)
Comment (mxmCherry) [reply lib/grape/app/templates/Rakefile]: Hint for rspec spec_helper:

```
  config.before :suite do
    ActiveRecord::Migration.maintain_test_schema!
  end
```

Comment (dim) [reply lib/grape/app/templates/Rakefile]: `if defined?(ActiveRecord)`

**Comment by mxmCherry** [comment]
I really don't know what to do with ruby 2.5.1 testing:

```
bundler: failed to load command: rake (/home/travis/build/bsm/grape-app/vendor/bundle/ruby/2.5.0/bin/rake)
```

No idea why does it seek for rake in 2.5.0

**Pull request merged**